custom_components/kumo/climate.py

```diff
@@ -128,8 +128,6 @@ class KumoThermostat(CoordinatedKumoEntity, ClimateEntity):
         "runstate",
     ]
 
-    _enable_turn_on_off_backwards_compatibility = False # can be removed once 2024.12 is no longer supported
```

Comment:
My understanding is this variable needs to remain set until the backwards compatibility code is removed. Otherwise we get a warning in the logs.

Reply:
As of https://github.com/home-assistant/core/commit/c41cf570d3f312ec3e1f5d0701b789d3e45771ff this variable is no longer used in the code. The change will be released on `2025.1` so the beta is Dec 25 and the live release is Jan 1.

From my read of the above code, what this variable would do is:
- when unset or `True` it would add the corresponding feature flags when turn_on/turn_off are defined or when `HVACMode.OFF` was in the modes
- when `False` it would do nothing (that's what we have here). The warning I saw on my installation was from trying to issue a `turn_off` command without a corresponding feature flag

Looking forward, the platform steers you to have (1) both the feature flag and turn_on/turn_off defined, or (2) neither. The PR tries to implement (1) since we are close to `2025.1` and so that we can have `turn_off` functionality particularly in voice assistants

_Please correct me if I misunderstood_

Reply:
Thought about this some more. While we are still on `2024.12` it's better to keep `_enable_turn_on_off_backwards_compatibility = False` to prevent `turn_on` from being added automatically. But regardless, implementing `turn_off` & adding the feature flag is fine in both versions